javaparser-core/src/main/java/com/github/javaparser/javadoc/description/JavadocInlineTag.java

```diff
@@ -101,7 +105,7 @@ public String getName() {
 
     @Override
     public String toText() {
-        return "{@" + tagName + this.content +"}";
+        return "{@" + tagName + " " + content + "}";
```

Comment:
Is there always just a space between the tagName and content? Is it acceptable to lose the precise original form or not?